app/models/user.rb

```diff
@@ -4,6 +4,6 @@ class User < ApplicationRecord
   has_many :likes
 
   def recent_posts
-    comments.order ('created_at DESC').limit(5)
+    posts.order ('created_at DESC').limit(5)
   end
```

Comment:
- [ ] Another error shows up in the users/show.html.erb file:

<img width="561" alt="image" src="https://user-images.githubusercontent.com/77190203/173191097-73474331-70a0-48db-b495-bed28ddf22a8.png">

Please remove the white space between "order" and "(" symbol.